core/tests/protractor-browserstack.conf.js

```diff
@@ -83,36 +85,36 @@ exports.config = {
     // in 'full' individually. This is to parallelize the suites on
     // browserstack.
     full: [
-      'protractor/accessibility.js',
-      'protractor/learnerFlow.js',
-      'protractor/libraryFlow.js',
-      'protractor_mobile/navigation.js',
-      'protractor/profileMenuFlow.js',
-      'protractor/subscriptionsFlow.js'
+      '../../local_compiled_js/core/tests/protractor/accessibility.js',
```

Comment:
Ditto -- does this work with --prod_env?

Reply:
I have added a compile option in build which compiles ts files to `local_compiled_js` whenever frontend or e2e test are run. This ensures that local_compiled_js is present irrespective of prod_mode when we are testing. So, yes it works in prod mode also.